lib/radiator/directory/editor/manager.ex

```diff
@@ -269,9 +300,14 @@ defmodule Radiator.Directory.Editor.Manager do
       {:error, %Ecto.Changeset{}}
   """
   def publish(subject = %type{}) when type in [Podcast, Episode, AudioPublication] do
-    subject
-    |> type.changeset(%{publish_state: :published})
-    |> Repo.update()
+    result =
+      subject
+      |> type.changeset(%{publish_state: :published})
+      |> Repo.update()
+
+    refresh_feed_cache(subject)
```

Comment:
This recurring call probably wants some clearer language concept around it, so it is not forgotten when it is needed.

Reply:
I'm not happy with it as well. That pattern where I have to keep the result in a tmp variable so I can still use it as the function return feels very awkward.

```
tmp = do_something()
refresh_feed_cache(subject)
tmp
```

Not sure what a clean solution would look like.